### ISSUE TITLE
Create a marker directive

### ISSUE THREAD

**Issue by blais** [beancount/beancount#258]
**[Original report](https://bitbucket.org/blais/beancount/issue/258) by Martin Blais (Bitbucket: [blais](https://bitbucket.org/blais), GitHub: [blais](https://github.com/blais)).**

----------------------------------------

Re. #257: Consider creating a "market" directive.

Fava has such a feature (and code), see the "insert-entry"-option here: https://fava.pythonanywhere.com/example-with-budgets/help/options/

Basically it works like this: You can add custom "insert-entry" entries with a RegEx, and when adding a transaction, the position where it should go in the file is determined by those custom entries.
